Source/ResponseSerialization.swift

```diff
@@ -559,7 +562,7 @@ extension DownloadRequest {
             guard let fileURL = fileURL else {
                 return .failure(AFError.responseSerializationFailed(reason: .inputFileNil))
             }
-
+            
```

Comment:
Unnecessary whitespace changes. You can run the `Cleanup Whitespace` target to eliminate these.